package.json

```diff
@@ -1,63 +0,0 @@
-{
```

Comment:
How come this file is being shown as deleted? I cloned your repo and the file is this there. 

Another bizarre one.